MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,659 @@
+    ...
+    <chart:SfCartesianChart.Annotations>
+        <chart:EllipseAnnotation X1="2" X2="4" Y1="10" Y2="15"/>
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var ellipse = new EllipseAnnotation()
+    {
+        X1 = 2,
+        Y1 = 10,
+        X2 = 4,
+        Y2 = 15,    
+    };
+
+    chart.Annotations.Add(ellipse);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Ellipse annotation in MAUI Chart](Annotation_image/Ellipse_annotation.png)
+
+N> When the [X2]() and [Y2]() properties of the [EllipseAnnotation]() are set, the [Height]() and [Width]() properties become ineffective.
+
+## Line annotation
+
+The [LineAnnotation]() is used to draw a line at specific points in the chart area.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+
+    <chart:SfCartesianChart.Annotations>
+        <chart:LineAnnotation X1="0.5" Y1="10" X2="3.5" Y2="20" Text="Line"/>
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    var line = new LineAnnotation()
+    {
+        X1 = 0.5,
+        Y1 = 10,
+        X2 = 3.5,
+        Y2 = 20,
+        Text = "Line"
+    };
+
+    chart.Annotations.Add(line);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+   
+![Line Annotation in MAUI Chart](Annotation_image/Line_annotation.png)
+
+## Vertical and Horizontal line annotation
+
+The [VerticalLineAnnotation]() and [HorizontalLineAnnotation]() are used to draw vertical and horizontal lines at specific points in the chart area.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+        <chart:SfCartesianChart.Annotations>
+            <chart:VerticalLineAnnotation X1="2"/>
+            <chart:HorizontalLineAnnotation Y1="20"/>
+        </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var vertical = new VerticalLineAnnotation()
+    {
+        X1 = 2,
+    };
+
+    var horizontal = new HorizontalLineAnnotation()
+    {
+        Y1 = 20,
+    };
+
+    chart.Annotations.Add(vertical);
+    chart.Annotations.Add(horizontal);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation in MAUI Chart](Annotation_image/Horizontal_vertical_annotation.png)
+
+## Displaying axis label for vertical and horizontal line annotations
+
+The [VerticalLineAnnotation]() and [HorizontalLineAnnotation]() display the axis labels where the line is placed. The [ShowAxisLabel]() property is used to show or hide the axis’s label. This feature can be enabled by setting the [ShowAxisLabel]() property to true, as shown in the code snippet below:
```

Comment:
The VerticalLineAnnotation and HorizontalLineAnnotation allow for the display of axis labels at the line's position using the ShowAxisLabel property. The default value of the ShowAxisLabel property is "False."

Reply:
Updated the content